src/main/java/com/minecolonies/coremod/tileentities/TileEntityColonyBuilding.java

```diff
@@ -5,25 +5,47 @@
 import com.minecolonies.coremod.colony.ColonyView;
 import com.minecolonies.coremod.colony.buildings.AbstractBuilding;
 import com.minecolonies.coremod.colony.permissions.Permissions;
+import com.minecolonies.coremod.inventory.AbstractInteractiveItemStackHandler;
 import com.minecolonies.coremod.util.Log;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.nbt.NBTTagCompound;
 import net.minecraft.network.NetworkManager;
 import net.minecraft.network.play.server.SPacketUpdateTileEntity;
-import net.minecraft.tileentity.TileEntityChest;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.ITickable;
 import net.minecraft.util.math.BlockPos;
+import net.minecraftforge.items.CapabilityItemHandler;
+import net.minecraftforge.items.IItemHandler;
+import net.minecraftforge.items.ItemStackHandler;
 import org.jetbrains.annotations.NotNull;
 
 /**
  * Class which handles the tileEntity of our colonyBuildings.
  */
-public class TileEntityColonyBuilding extends TileEntityChest
+public class TileEntityColonyBuilding extends TileEntity implements ITickable
 {
     /**
-     * NBTTag to store the colony id.
+     * NBT tag definitions.
      */
-    private static final String TAG_COLONY = "colony";
-    private static final String TAG_MIRROR = "mirror";
+    private static final String TAG_COLONY      = "colony";
+    private static final String TAG_MIRROR      = "mirror";
+    private static final String TAG_CUSTOM_NAME = "CustomName";
+    private static final String TAG_INVENTORY   = "inventory";
+
+    /**
+     * The item handler.
+     */
+    private final ItemStackHandler itemHandler = new AbstractInteractiveItemStackHandler(27) {
+        @Override
+        public boolean isUseableByPlayer(EntityPlayer player) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)